.github/CODEOWNERS

```diff
@@ -23,6 +23,12 @@
 /.pre-commit-config.yaml                @DataDog/agent-devx
 /.vscode/                               @DataDog/agent-devx
 
+/.bazel*                                @DataDog/agent-build
+/MODULE.bazel*                          @DataDog/agent-build
+# Temporary during Bazel migration
+/**/BUILD.bazel                         @DataDog/agent-build
+/**/*.bzl                               @DataDog/agent-build
```

Comment:
Apparently we have some linting that doesn't allow these to point to non existing files:

https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/1070644686

```
$ dda inv -- -e github.lint-codeowner
The following rules are outdated: they don't point to existing file/directory
	- /**/*.bzl	[('TEAM', '@DataDog/agent-build')]
	- /**/BUILD.bazel	[('TEAM', '@DataDog/agent-build')]
	- /MODULE.bazel*	[('TEAM', '@DataDog/agent-build')]
	- /.bazel*	[('TEAM', '@DataDog/agent-build')]
```

Reply:
Yes. I'm inclined to skip the test because the dangling issue should be a warning. Now that the MODULE.bazel PR is in, I can make this pass.